pkg/diffs/diffs_test.go

```diff
@@ -0,0 +1,132 @@
+package diffs
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	prowconfig "k8s.io/test-infra/prow/config"
+)
+
+const orgRepo = "org/repo"
+
+func TestGetChangedPresubmits(t *testing.T) {
+	basePreSubmit := makeBasePresubmit()
+	testCases := []struct {
+		name                  string
+		masterConfigGenerator func() (config *prowconfig.Config)
```

Comment:
I think one func is cleaner -- you're always going to be taking A and changing it to get B, with two you must enforce that they both start with A